pleque/io/compass.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+from pleque import Equilibrium
+
+
+def read_fiesta_equilibrium(filepath, first_wall=None):
+    """
+    Current versions of the equilibria are stored in
+    `/compass/Shared/Common/COMPASS-UPGRADE/RP1 Design/Equilibria/v3.1`
+
+    :param filepath: Path to fiesta g-file equilibria
+    :param first_wall: Path to datafile with limiter line. (Fiesta doesn't store limiter contour into g-file).
+        If `None` IBA limiter v 3.1 is taken.
+    :return: Equilibrium: Instance of `Equilibrium`
+    """
+    from pleque.io.readgeqdsk import readeqdsk_xarray
+    from scipy.interpolate import UnivariateSpline
+    import pkg_resources
+
+    resource_package = __name__
+
+    if first_wall is None:
```

Comment:
I would make this ready for time, when fieste output includes some first wall coordinates. Include a check if the z_lim and r_lim are empty and if they are load it from an external file.

Reply:
Done